packages/jaeger-ui/src/components/TracePage/TracePageHeader/SpanGraph/index.tsx

```diff
@@ -31,18 +32,6 @@ type SpanGraphProps = {
   updateNextViewRangeTime: (nextUpdate: ViewRangeTimeUpdate) => void;
 };
 
-/**
- * Store `items` in state so they are not regenerated every render. Otherwise,
- * the canvas graph will re-render itself every time.
- */
-type SpanGraphState = {
```

Comment:
shouldn't this type declaration remain and used in `getItem`?

Reply:
that's a great call out. I'll re-add these types.

Reply:
Added a new type `SpanItem` and documented it as the return of the getItem(s) functions.